hpy/devel/__init__.py

```diff
@@ -112,6 +112,9 @@ def __bootstrap__():
     ext_filepath = pkg_resources.resource_filename(__name__, {ext_file!r})
     m = load_from_spec(Spec({module_name!r}, ext_filepath))
     m.__file__ = ext_filepath
+    m.__loader__ = __loader__
```

Comment:
I'm not 100% sure that copy the loader and spec are the perfect thing to do, but:

1. These are the loader and spec that were used to import the module, so they're correct in that sense.
2. There isn't another loader and spec than can be used to import an hpy.universal extension.